x-pack/test_serverless/functional/test_suites/common/discover/x_pack/visualize_field.ts

```diff
@@ -36,11 +33,20 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   }
 
   describe('discover field visualize button', () => {
-    beforeEach(async () => {
+    before(async () => {
+      // TODO: Serverless tests require login first
+      // Security project requires admin role, search/oblt project passes with developer/editor.
+      await PageObjects.svlCommonPage.loginAsAdmin();
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
       await kibanaServer.importExport.load(
         'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
       );
+      await kibanaServer.uiSettings.replace({
+        defaultIndex: 'logstash-*',
+      });
+    });
```

Comment:
This test was a real pain to migrate from base auth default user to SAML role: different roles definitions in 3 projects ('Editor' for Search is not good enough), then default data view was not the one test is using.